static/js/generic_multiple_choice.html

```diff
@@ -0,0 +1,288 @@
+<html>
```

Comment:
Initial file uses tabs and it will mean I change whole file just for improving style that is bad practice.